src/main/java/org/folio/rest/impl/TitlesAPI.java

```diff
@@ -27,8 +54,18 @@ public void getOrdersStorageTitles(String query, String totalRecords, int offset
 
   @Override
   @Validate
-  public void postOrdersStorageTitles(Title entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(TITLES_TABLE, entity, okapiHeaders, vertxContext, OrdersStorageTitles.PostOrdersStorageTitlesResponse.class, asyncResultHandler);
+  public void postOrdersStorageTitles(Title title, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    pgClient.withConn(conn -> titleService.saveTitle(title, conn)
```

Comment:
We should close connection after usage, not after providing response, so need to add additional ')' here